mailbox/hbase/src/main/java/org/apache/james/mailbox/hbase/mail/model/HBaseMailbox.java

```diff
@@ -22,8 +22,10 @@
 
 import org.apache.james.mailbox.hbase.HBaseId;
 import org.apache.james.mailbox.model.MailboxACL;
+import org.apache.james.mailbox.model.MailboxAnnotation;
 import org.apache.james.mailbox.model.MailboxPath;
 import org.apache.james.mailbox.model.SimpleMailboxACL;
+import org.apache.james.mailbox.model.SimpleMailboxAnnotation;
```

Comment:
Why do we need these imports ?

```
import org.apache.james.mailbox.model.SimpleMailboxAnnotation;
```

Will fail as class no more exists => Continuous integration wil fail.